modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,290 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a hosted monitoring and log storage service. You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+You can include a specific CloudWatch topic in the output or use the default. The CloudWatch output can use a TCP (insecure) or TLS (secure TCP) connection.
+
+.Procedure
+
+. Create a `Secret` YAML file similar to the following example. Use base64-encoded values for the `username` and `password` fields. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cw-secret
+  namespace: openshift-logging
+data:
+  aws_access_key_id: F1lAUtMIGg2LzANQkVIWVhSRFQKQ
+  aws_secret_access_key: aSEvRkovGEpPMkxZNHhWK1dwb2p0UY1kvHlNdEt6Z0VhNHBSV1J3Lgo=
+----
```

Comment:
```
  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
```
These decode to Amazon's own examples in https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html :)

Reply:
Updated. Thanks.